custom_components/ocpp/switch.py

```
@@ -27,7 +30,7 @@
     DOMAIN,
     ICON,
 )
-from .enums import HAChargerServices, HAChargerStatuses
+from .enums import ConfigurationKey, HAChargerServices, HAChargerStatuses
```

Ruff (0.16.0)

[error] Module level import not at top of file

(E402)